src/components/call/CallGeneralInfo.tsx

```diff
@@ -53,6 +74,62 @@ const CallGeneralInfo: React.FC<{
     }
   }, [startCall, endCall, formik]);
 
+  function validateRefnumber(input: string) {
+    let error;
+    const regExp = /((([a-z|\d]+)){)({digits:[1-9]+})/;
+    const prefix = /((([a-z|\d]+)){)/;
+    const suffix = /({digits:[1-9]+})/;
+    if (input) {
+      if (input.match(regExp)) {
+      } else if (!input.match(suffix)) {
+        error = 'Invalid parameter.';
+      } else if (!input.match(prefix)) {
+        error = 'Invalid parameter.';
+      }
+    }
+
+    return error;
+  }
```

Comment:
I notice you've replaced having `Invalid Suffix` and `Invalid Prefix` with just `Invalid parameter` for both.
Was this intentional? If so, I think the code can be simplified a little:
```suggestion
  function validateRefnumber(input: string) {
    const regExp = /((([a-z|\d]+)){)({digits:[1-9]+})/;
    if (input && !input.match(regExp)) {
      return 'Invalid parameter.';
    }
  }
```
Please check this works before committing 😅

Reply:
@EdwardHaynes Thank you so much for having a look at this. 😊
I misunderstood what @simonfernandes mentioned before about the error messages and didn't update the code properly. He suggested displaying `Invalid prefix` and `Invalid parameter` for the invalid suffix. I need to make those changes.